### ISSUE TITLE
Batavia testserver appears to be broken in Safari 9.1.3

### ISSUE THREAD

**Issue by ilankham** [pybee/batavia#520]
Here's the output I saw in the Console:

```
[Error] TypeError: data.slice is not a function. (In 'data.slice(0, 4)', 'data.slice' is undefined)
	run (batavia.js:25114)
	bootstrap (127.0.0.1:474)
	onload (127.0.0.1:495)
```

**Comment by lschumm** [comment]
@ilankham Please provide more information on your issue. I've run it in Safari, and have not had a similar issue. Are you sure you've followed all the instructions verbatim?

**Comment by ilankham** [comment]
After following http://batavia.readthedocs.io/en/latest/intro/tutorial-0.html to download and build Batavia, I was able to run the testserver using Django version 1.10 and execute Python code in Chrome 58.0.3029.110.

However, the above error occurred when accessing the testserver in Safari 9.1.3 and attempting to execute the Python code `print("Hello, World")`. In addition, no console output was generated.

**Comment by lschumm** [comment]
Ah, I didn't read your Safari version. Safari 9 doesn't support the `.slice()` method; code should be changed to implement a workaround. Similar issue: https://github.com/crocodilertc/crocodile-rtc/issues/15.

**Comment by ilankham** [comment]
This issue has been addressed in pull request #527

**Comment by ilankham** [comment]
The testserver now appears to work as expected in Safari 9, so this issue is being closed.
